# timestamp information
uint64 framenumber
time stamp # start of image frame acquisition timestamp

TrackedObject[] tracked_objects
